src/models/searchservice/response/directanswer/FeaturedSnippetDirectAnswer.ts

```diff
@@ -1,18 +1,9 @@
-import { DirectAnswer } from './DirectAnswer';
-import { DirectAnswerType } from './DirectAnswerType';
-import { Snippet } from '../Snippet';
-import { BuiltInFieldType } from './BuiltInFieldType';
-
+import { MultiLineTextFeaturedSnippetDirectAnswer } from './MultiLineTextFeaturedSnippetDirectAnswer';
+import { RichTextFeaturedSnippetDirectAnswer } from './RichTextFeaturedSnippetDirectAnswer';
 /**
- * A direct answer which was found within a document.
+ * Possible built-in {@link BaseFeaturedSnippetDirectAnswer} interfaces
```

Comment:
Can we change "Possible" to "All possible" (or something similar to show completeness)?